proto/v1/cvss.proto

```diff
@@ -157,3 +157,10 @@ message CVSS {
     IMPACT_NONE = 3;
   }
 }
+
+// CVSS Version.
+enum CVSSVersion {
+  CVSS_VERSION_UNSPECIFIED = 0;
+  CVSS_VERSION_2 = 2;
```

Comment:
Is there a reason you're starting from 2? This is just a proto field, so is there any place this value would get exposed as 2?

Reply:
Thanks for catching this. Corrected.